console/tsconfig.json

```diff
@@ -14,9 +14,13 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "importHelpers": true,
-    "target": "es2020",
+    "target": "ES2022",
     "module": "es2020",
-    "lib": ["es2020", "dom"]
+    "lib": [
+      "es2020",
+      "dom"
```

Comment:
This looks weird mixing es2020 and ES2020

Reply:
changed both to ES2022